### PR TITLE
Updated Export-DbaUser to include SQL 2019

### PR DESCRIPTION
**Pull request by dbaduck**
Updated ValidateSet, ScriptVersion and fixed a typo.

## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [X] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
Add in SQLServer2019 into the ValidateSet and in the script version variables used.

### Commands to test
Export-DbaUser

### CODE REVIEW

File: functions/Export-DbaUser.ps1 [hunk 1]
```diff
@@ -159,7 +159,8 @@ function Export-DbaUser {
             'SQLServer2012'        = 'Version110'
             'SQLServer2014'        = 'Version120'
             'SQLServer2016'        = 'Version130'
-            'SQLServer2017'        = 'Version140'
+			'SQLServer2017'	       = 'Version140'
+			'SQLServer2019'        = 'Version150'
```
Comment (wsmelton) [inline functions/Export-DbaUser.ps1:163]: Looks like formatting is off for some reason...

File: functions/Export-DbaUser.ps1 [hunk 2]
```diff
@@ -169,7 +170,8 @@ function Export-DbaUser {
             'Version110' = 'SQLServer2012'
             'Version120' = 'SQLServer2014'
             'Version130' = 'SQLServer2016'
-            'Version140' = 'SQLServer2017'
+			'Version140' = 'SQLServer2017'
+			'Version150' = 'SQLServer2019'
```
Comment (wsmelton) [inline functions/Export-DbaUser.ps1:174]: Please update formatting....

**Comment by wsmelton** [comment]
Test failed due to the formatting...

**Comment by dbaduck** [comment]
I updated the formatting. For some reason it was using tabs and should have been spaces.

**Comment by dbaduck** [comment]
Still getting the hang of jumping in again, do I need to close this and open a new pull request?  Or how do I get the changes looked at?  Thanks Shawn.

**Pull request merged**